TreeOutputLib/TreeOutput.py

```diff
@@ -138,13 +138,15 @@ def checkRequiredKey(self, key, args):
     #  For each timestep a file is created throughout the simulation.
     #  This function is only able to work, if the output directory exists and
     #  is empty at the begin of the model run
-    def writeOutput(self, tree_groups, time):
+    def writeOutput(self, tree_groups, time, it_is_last_ts=False):
```

Comment:
After some thoughts I would suggest a more descriptive name for the keyword.
```suggestion
    def writeOutput(self, tree_groups, time, force_output=False):
```

Reply:
good idea